lib/x11utils.pm

```diff
@@ -233,6 +233,14 @@ sub handle_login {
     assert_screen 'displaymanager-password-prompt';
     type_password($mypwd);
     send_key 'ret';
+    wait_still_screen;
+    # for S390x testing, since they are not using qemu built-in vnc, it is
+    # expected that polkit authentication window can open for first time login.
+    # see bsc#1177446 for more information.
+    if (check_screen(qw[authentication-required-user-settings authentication-required-modify-system])) {
```

Comment:
You need to give a timeout value to check_screen, otherwise, you will get errors
 `Argument "authentication-required-modify-system" isn't numeric in multiplication (*) at /usr/lib/os-autoinst/bmwqemu.pm line 324.`
See https://openqa.suse.de/tests/6935288/file/autoinst-log.txt

Reply:
Besides, have you tried the `handle_additional_polkit_windows` (https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/234b496213ed9bc282e0279362aa8d32ed85088c/lib/opensusebasetest.pm#L971) ?

Reply:
Thanks, I'll change `check_screen`.

About `handle_additional_polkit_windows`, it will introduce extra softfailure, and it is too heavy, so not very suitable here.